### PR TITLE
inform the user that some upgrade steps are repeatable after an error

### PR DESCRIPTION
**Pull request by MaximilianMeister**
repeatable_upgrade_error

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
cc @jsuchome @vuntz @kwwii

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 1]
```diff
@@ -561,6 +561,7 @@ en:
     upgrades:
       title: 'Upgrade'
       header: '%{product}: Upgrade from SUSE OpenStack Cloud 5'
+      repeat_after_conflict: 'Please repeat this step after resolving the conflicts'
```
Comment (jsuchome) [inline crowbar_framework/config/locales/crowbar/en.yml]: The dot at the end of sentence is missing. Also, I don't think "Please" is necessary here (but some English speaker should be judge of that).

**Comment by jsuchome** [comment]
In general, I think it is not necessary (as I tried to explain in the bug report). There's nothing else user can do in the UI than repeating the step anyway, so this adds no extra value.

(This is not -1, I just think it is not needed. But if others think extra message helps, I agree that this is the simple way to add it)

**Comment by MaximilianMeister** [comment]
I have updated the display now after talking to @kwwii, @jsuchome he also said it would be good to at least give a hint about it

**Comment by jsuchome** [comment]
This looks good actually. +1

**Comment by vuntz** [comment]
I am a bit confused: why is the string talking about conflicts instead of errors?

**Comment by MaximilianMeister** [comment]
> I am a bit confused: why is the string talking about conflicts instead of errors?

Here Errors is better I agree, will change it. But in https://github.com/crowbar/crowbar-core/pull/386 i would rather stick to conflicts

**Comment by MaximilianMeister** [comment]
changed to `Errors`

**Comment by vuntz** [comment]
+1

**Comment by jsuchome** [comment]
Still +1

**Comment by rsalevsky** [comment]
Gating failed in tempest.

**Pull request merged**